olap/rolap/dbmapper/mondrian/src/test/resources/WrongFoodMart.xml

```diff
@@ -0,0 +1,827 @@
+<?xml version="1.0"?>
+<Schema name="FoodMart">
```

Comment:
could you use a much simpler example that provicates the validation error?

Reply:
Yes. I plan to do that when validation will work